### ISSUE TITLE
The content of “Favorite color” tab cannot be read by screen reader

### ISSUE THREAD

**Issue by mathisscott** [vmware/clarity#1943]
URL: http://clarity-ks.surge.sh/datagrids

Reproduced Steps:
Go to Datagrids page
Use screen reader to read the “Favorite color” tab (refer to “Screen shot”) in the page

Actual Results:
The “Favorite color” tab cannot be read by screen reader

Expected Results:
The content of “Favorite color” should be read by screen reader correctly.

Violation of Section 508 Standards:
1194.22(n) When electronic forms are designed to be completed on-line, the form shall allow people using assistive technology to access the information, field elements, and functionality required for completion and submission of the form, including all directions and cues. 
 
User Impact: Blind and disable with hand.

![image.png](https://images.zenhubusercontent.com/582f4cffb9dc3c6220222f57/c434882b-e0f7-4eb9-9540-7765341e9cbf)

NOTE: Implementation issue

**Comment by mathisscott** [comment]
Demo content.
